@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStorageUsage.vue

```diff
@@ -5,11 +5,8 @@
       :right="$t('top-#', { n: N_ITEMS })"
     />
     <NoDataError v-if="hasError" />
-    <UsageBar
-      v-else
-      :data="isReady ? data.result : undefined"
-      :nItems="N_ITEMS"
-    >
+    <UiCardSpinner v-else-if="isFetching" />
+    <UsageBar v-else :data="isReady ? data.result : []" :nItems="N_ITEMS">
```

Comment:
What do you think of this?

```suggestion
    <UiCardSpinner v-else-if="!isReady" />
    <UsageBar v-else :data="data.result" :nItems="N_ITEMS">
```